### PR TITLE
[functionapp] Remove deprecated devops-build command

### PR DESCRIPTION
**Pull request by Hazhzeng**
The `az functionapp devops-build` has been deprecated for more than 2 weeks and suppose to be removed in version **2.0.68 (Sprint 64)**.

This PR removed the command and suppose to be out in Sprint 64.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by Hazhzeng** [comment]
@ahmedelnably

**Comment by marstr** [comment]
Howdy @Hazhzeng, I've gone ahead and merged your changes with the most recent version of dev to fix some conflicts for you. Other than having your local repository need a pull there should be no further action needed by you.

**Comment by Hazhzeng** [comment]
Much appreciated @marstr

**Comment by Hazhzeng** [comment]
@yugangw-msft Thanks for reminding. The HISTORY.rst is up to date now.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ Release History
 
 * Update ADLS version (0.0.45).
 
+**Appservice**
+
+* functionapp: remove deprecated `az functionapp devops-build` command
```
Comment (tjprescott) [inline src/azure-cli/HISTORY.rst]: This needs to be annotated as a breaking change.

Comment (Hazhzeng) [reply src/azure-cli/HISTORY.rst]: @tjprescott 👍 Fixed

**Pull request merged**